### PR TITLE
wp details overview cleanup

### PR DESCRIPTION
**Pull request by 0xF013**
[`* `#17684` overview cleanup`](http://community.openproject.org/work_packages/17684)
- [x] protractor failing
- [x] refactor protractor tests

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage. <a href='http://0xf013-feature-overview-cleanup.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Click to open.</a>

**Comment by 0xF013** [comment]
amended

**Comment by myabc** [comment]
@0xF013 related to this (and as just mentioned on the call), I'd like to change the structure of the in-place editing directives: https://gist.github.com/myabc/e855281c3a68bcb1f974

**Comment by myabc** [comment]
@0xF013 marking WIP, as I think you're still working on this (?)

**Comment by 0xF013** [comment]
@myabc thanks, should be ready now

File: frontend/public/templates/work_packages/tabs/overview.html [hunk 1]
```diff
@@ -26,64 +26,41 @@ <h3 ng-bind="I18n.t('js.work_packages.property_groups.' + group.groupName)"></h3
...
-                ined-type="select2"
-                ined-entity="workPackage"
-                ined-attribute="{{propertyData.property}}"
-                ined-attribute-embedded="false"
-                placeholder="-">
-              </div>
+        <div ng-if="!inplaceProperties[propertyData.property]"
+          ng-class="['work-package-attributes--value', '-' + propertyData.format]">
+          <empty-element ng-if="propertyData.value === null"></empty-element>
+          <div ng-if="propertyData.value !== null" ng-switch="propertyData.format">
```
Comment (myabc) [inline frontend/public/templates/work_packages/tabs/overview.html:44]: @0xF013 why can't we just test for falseyness here?

Comment (0xF013) [inline frontend/public/templates/work_packages/tabs/overview.html:44]: @myabc I noticed some values can be zero, like percent done, but I wasn't sure it means it is an empty attribute

Comment (myabc) [inline frontend/public/templates/work_packages/tabs/overview.html:44]: @0xF013 then we might need `propertyData.value !== null && propertyData.value !== undefined` or use negated double equals operator: `propertyData.value != null`

Comment (0xF013) [inline frontend/public/templates/work_packages/tabs/overview.html:44]: @myabc indeed. I think we should check if falsy means empty.

Comment (0xF013) [inline frontend/public/templates/work_packages/tabs/overview.html:44]: @myabc I remembered now why I used the strict comparison. Previously it was ng-switch-when="null" which does a strict comparison

**Pull request merged**